ui/src/components/Popover.vue

```diff
@@ -214,4 +227,7 @@ export default class Popover extends Vue {
   visibility: visible;
   z-index: 2;
 }
+.weaverbird-popover--calculated-height {
```

Comment:
Not sure to understand the commentary... The subchild is not applied without declaring the css class here? It's strange... It's should works without it no :thinking: ?

Reply:
Or it is just a commentary to notify devs that they should not remove the class herself? On the component ?

Reply:
I worry that someone see the unused class in the file and delete it 😅 
but maybe I'm a bit paranoid 🤣 
I can delete it

Reply:
At the time I wrote it, it sounded clear to me, but in review you're right, it's weird, I'm going to delete this